.eslintrc.js

```diff
@@ -39,13 +39,15 @@ module.exports = {
     'jsdoc/require-description': 'warn',
     'jsdoc/require-param': 'warn',
     'jsdoc/require-returns': 'warn',
-    'jsdoc/check-tag-names': 'warn',
+    'jsdoc/check-tag-names': ['warn', {
+      definedTags: ['json']
+    }],
     'jsdoc/check-alignment': 'warn',
     'jsdoc/newline-after-description': 'off', // does not work with 'use strict'
-    'jsdoc/check-indentation': 'warn',
     'jsdoc/lines-before-block': 'warn',
     'jsdoc/check-types': 'warn',
-    'jsdoc/require-hyphen-before-param-description': 'warn'
+    'jsdoc/require-hyphen-before-param-description': 'warn',
+    'jsdoc/check-indentation': ['warn', { excludeTags: ['json', 'example'] }]
```

Comment:
This custom tag is breaking the docs due to this being set - 
```
    // We want JSDoc to error if it encounters an unknown tag
    allowUnknownTags: false,
```

We can either use the default @example tag everywhere or turn this on. 

The other option is to write a pulgin for jsdoc which is not necessary i feel.